### PR TITLE
Plugin mute_inputs: added posibility to enable/disable inputs and previews

### PR DESCRIPTION
**Pull request by dpeite**
Added a new plugin and new rest endpoints to enable/disable inputs and previews

### CODE REVIEW

File: galicaster/recorder/recorder.py [hunk 1]
```diff
@@ -80,6 +81,10 @@ def __init__(self, bins, players={}):
                 self.bins[name] = Klass(bin)
                 self.pipeline.add(self.bins[name])
                 
+                self.mute_status["input"][name] = True
```
Comment (hcaloto) [inline galicaster/recorder/recorder.py]: Use enable_preview and enable_input instead of these sentences

File: galicaster/recorder/recorder.py [hunk 2]
```diff
@@ -46,7 +46,8 @@ def __init__(self, bins, players={}):
         self.dispatcher = context.get_dispatcher() 
         self.players = players
         self.restart = False
-        self.mute = False
+#        self.mute = False
+        self.mute_status = {"input":{},"preview":{}}
```
Comment (hcaloto) [inline galicaster/recorder/recorder.py]: Add a function to return self.mute_status

File: galicaster/classui/mainwindow.py [hunk 3]
```diff
@@ -164,6 +164,28 @@ def insert_page(self, page, label, cod):
         """Insert an area on the main window notebook widget"""
         self.nbox.insert_page(page, Gtk.Label(label=label), cod)
 
+    def insert_button(self, button, ui, box_id, **kwargs):
+        try:
+            builder = self.nbox.get_nth_page(ui).gui
+        except Exception as error:
+            self.logger.error("La vista no existe")
```
Comment (rubenrua) [inline galicaster/classui/mainwindow.py]: Use English.

File: galicaster/classui/mainwindow.py [hunk 4]
```diff
@@ -164,6 +164,28 @@ def insert_page(self, page, label, cod):
         """Insert an area on the main window notebook widget"""
         self.nbox.insert_page(page, Gtk.Label(label=label), cod)
 
+    def insert_button(self, button, ui, box_id, **kwargs):
```
Comment (rubenrua) [inline galicaster/classui/mainwindow.py]: I miss a small documentation. :) (See python docstrings).

**Pull request merged**